README.md

```diff
@@ -46,6 +46,28 @@ $ kubectl apply -f https://raw.githubusercontent.com/mittwald/kubernetes-replica
 
 ## Usage
 
+### Role and rolebinding replication
+
+In order to replicate roles and rolebindings, the service account tied to the replicator needs to have the correct permissions. The chart currently offers two options to accomplish this:
+
+- Set the value `grantClusterAdmin`to `true`, which grants the service account admin privileges. This is set to `false` by default, as having a service account with that level of access might be undesirable due to the potential security risks attached. 
+
+- Set the lists of needed api groups and resources explicitely. The values `serviceAccount.apiGroups` and `serviceAccount.resources`, you can specify what priiliges the replicator is allowed to replicate.
+  
+  Example:
+
+  ```yaml
+  serviceAccount:
+  create: true
+  annotations: {}
+  name:
+  apiGroups: [ "", "apps", "batch", "extensions" ] 
+  resources: ["secrets", "configmaps", "roles", "rolebindings",
+     "cronjobs", "deployments", "events", "ingresses", "jobs", "pods", "pods/attach", "pods/exec", "pods/log", "pods/portforward", "services"]
```

Comment:
This would grant access on non-existent types like `batch/secrets` -- mainly a cosmetical issue (I hope), but maybe we could restructure this to a list of `apiGroup`/`resources` combinations?